test/functional/neutronless/disconnected_service/test_disconnected_service_creation.py

```diff
@@ -608,6 +588,39 @@ def create_mock_rpc_plugin():
 
     assert post_destroy_uris == empty_set
 
+def test_featureoff_grm_listener(setup_l2adjacent_test, bigip):
+    def create_mock_rpc_plugin():
```

Comment:
You could move this function into a fixture that can be used by other tests of a similar nature, if there are any.